src/Component/Cards/Card.scss

```diff
@@ -0,0 +1,22 @@
+.product_card_box {
+    .card {
+        .card-body {
+
+            p {
+                overflow: hidden;
+                text-overflow: ellipsis;
+                display: -webkit-box;
+                -webkit-line-clamp: 3;
+                /* number of lines to show */
+                line-clamp: 3;
+                -webkit-box-orient: vertical;
```

Comment:
If we are using SCSS, we don't need to use prefixes, webpack will take care of that.